packages/fxa-settings/package.json

```diff
@@ -26,14 +26,14 @@
       "src/**/*.{ts,tsx,js}",
       "!src/**/*.stories.*",
       "!src/**/mocks.*",
-      "!src/components/*(ScrollToTop|UnitRowRecoveryKey|PageAvatar|App)/*"
+      "!src/components/settings/*(ScrollToTop|UnitRowRecoveryKey|PageAvatar|App)/*"
```

Comment:
Probably needs to be `Settings`

Reply:
It does! Thank you I forgot, updating now!